test/src/cpp-integration-query-condition.cc

```diff
@@ -1610,6 +1610,82 @@ TEST_CASE(
   }
 }
 
+TEST_CASE(
+    "Testing sparse query condition with the same fragment domain.",
+    "[query][query-condition][sparse][same-fd]") {
```

Comment:
Please consider setting up new tests with REST infra and [rest] tag so that they are also run on REST-CI and test `tiledb://` arrays. 
It's normally low effort, documented here: https://github.com/TileDB-Inc/TileDB-Internal?tab=readme-ov-file#add-a-test-to-rest-ci

Reply:
Will do, thanks.